File: JAVA/Pila/Carro.java
package JAVA.Pila;

public class Carro {
    private String marca;
    private String modelo;
    private String color;
    private String placa;

    public Carro(String marca, String modelo, String color, String placa) {
        this.marca = marca;
        this.modelo = modelo;
        this.color = color;
        this.placa = placa;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public String toString() {
        return "*****************"+"\nMarca: " + marca + "\nModelo: " 
        + modelo + "\nColor: " + color + "\nPlaca: " 
        + placa+"\n*****************";
    }
}

File: C#/Pila/Carro.cs
public class Carro {
    string marca;
    string modelo;
    string color;
    string placa;

    public Carro(string marca, string modelo, string color, string placa) {
        this.marca = marca;
        this.modelo = modelo;
        this.color = color;
        this.placa = placa;
    }

    public string getMarca() {
        return this.marca;
    }

    public string getModelo() {
        return this.modelo;
    }

    public string getColor() {
        return this.color;
    }

    public string getPlaca() {
        return this.placa;
    }

    public void setMarca(string marca) {
        this.marca = marca;
    }

    public void setModelo(string modelo) {
        this.modelo = modelo;
    }

    public void setColor(string color) {
        this.color = color;
    }

    public void setPlaca(string placa) {
        this.placa = placa;
    }

    public string mostrar() {
        return "*****************"+"\nMarca: " + marca + "\nModelo: " 
        + modelo + "\nColor: " + color + "\nPlaca: " 
        + placa+"\n*****************";
    }
}

